plugins/@grouparoo/spec-helper/src/lib/specHelper.ts

```diff
@@ -124,7 +124,7 @@ export namespace helper {
   export const longTime = 60 * 1000; // 60 seconds
   export const mediumTime = 20 * 1000; // 20 seconds
   export const shortTime = 10 * 1000; // 10 seconds
-  export const defaultTime = process.env.CI ? mediumTime : shortTime;
+  export const defaultTime = process.env.CI ? mediumTime : longTime;
```

Comment:
Why the change?  You can tell a specific test of describe to have a longer timeout if needed